assignments/arrays.js

```diff
@@ -63,34 +75,82 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+function carId(obj) {
+  for (let index = 0; index < inventory.length; index++) {
+    if (inventory[index].id === 33) {
+      console.log(
+        `Car 33 is a ${inventory[index].car_year} ${
+          inventory[index].car_make
+        } ${inventory[index].car_model}`
+      );
+    }
+  }
+}
+
+carId(inventory);
 
+// // ==== Challenge 2 ====
+// // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 
+function last_car(arr) {
+  let lastCar = 0;
 
-// ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+  for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === lastCar) {
```

Comment:
did you test this one? How do you find the last index of an array. I'll give you a hint: how does a for loop know when to stop>?